_posts/0000-01-02-NusMes.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!
+---
+Your text
```

Comment:
**This is my new text and it's much** *more cool* than the one that was here before.